integrations/spark/openhouse-spark-runtime/src/main/antlr/com/linkedin/openhouse/spark/sql/catalyst/parser/extensions/OpenhouseSqlExtensions.g4

```diff
@@ -150,6 +172,8 @@ DATABASE: 'DATABASE';
 SHOW: 'SHOW';
 GRANTS: 'GRANTS';
 PATTERN: 'PATTERN';
+CLUSTER: 'CLUSTER';
+INTERVAL: 'INTERVAL';
```

Comment:
Do we plan to support at day level granularity like min 1 day and upto max 3 days?

Reply:
right now we should only be supporting hour level granularity

Reply:
Sounds good. We can add at day level granularity in future.